tests/baselines/reference/jsDeclarationsClasses.js

```diff
@@ -577,10 +577,6 @@ export class N<T> extends L {
  * @extends {N<U>}
  */
 export class O<U> extends N<U> {
-    /**
-     * @param {U} param
-     */
-    constructor(param: U);

```

Comment:
Why is this elided? source has constructor signature for O at https://github.com/microsoft/TypeScript/pull/41767/files#diff-8cef264cf87057b69d50eeda6098081c028ab139d5c75afecc19512c8e92e824R178

Reply:
It's identical to the base signature (the base signature is instantiated into the same signature as in the base class), and thus redundant.

Reply:
Interesting.. i thought that would be elided only if O had elided it.

Reply:
Nah - the node serializer looks at semantic information usually, not syntactic. Sometimes we reuse input type nodes where possible, but generally not wholesale signature declarations like this. So since the class only has one signature, and it's identical to the signature provided by the base class, we "elide" it so as to not include the base class's signature in all subclasses, since we don't need it.